frontend/src/component/project/Project/ProjectStatus/ProjectActivity.tsx

```diff
@@ -0,0 +1,90 @@
+import { useRequiredPathParam } from 'hooks/useRequiredPathParam';
+import { useProjectStatus } from 'hooks/api/getters/useProjectStatus/useProjectStatus';
+import ActivityCalendar, { type ThemeInput } from 'react-activity-calendar';
+import type { ProjectActivitySchema } from '../../../../openapi';
+import { styled, Tooltip } from '@mui/material';
+
+const StyledContainer = styled('div')(({ theme }) => ({
+    gap: theme.spacing(1),
+}));
+
+type Output = { date: string; count: number; level: number };
+
+export function transformData(inputData: ProjectActivitySchema): Output[] {
```

Comment:
This can be cleaned up later,  but basically for each day we need a level(color), and this divides them evenly into level buckets.